core/views.py

```diff
@@ -1,16 +1,22 @@
 from django.shortcuts import render
+from django.db.models.functions import TruncDay
+from django.db.models import Count
```

Comment:
Unused import

Reply:
thanks! I didn't cover all of the cool IDE tools that are nice to have setup (like black and some form of static analysis), but this definitely could be caught automatically (and/or could be a warning at commit time..?)

@andrea749 feel free to delete the line, and/or we can approve this PR and do the cleanup afterward as well

Reply:
> we can approve this PR and do the cleanup afterward as well

agreed fwiw — I think this is in good shape!